### PR TITLE
Update dependencies and build instructions

### LINKED ISSUE #1027: Update dependencies, and Tor binaries
For version 2.2, we should update to Qt 5.13.1, as well as other pip dependencies. And right now we're getting Tor binaries from Tor Browser 8.0.8, but the latest Tor Browser is 8.5.5.

### PR DESCRIPTION
**Pull request by micahflee**
Resolves #1027

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Open a command prompt, cd to the onionshare folder, and install dependencies wit
 pip install -r install\requirements.txt
 ```
 
-Install the Qt 5.13.0 from https://www.qt.io/download-open-source/. I downloaded `qt-opensource-windows-x86-5.13.0.exe`. In the installer, you can skip making an account, and all you need `Qt` > `Qt 5.13.0` > `MSVC 2017 32-bit`.
+Install the Qt 5.13.0 from https://www.qt.io/offline-installers. I downloaded `qt-opensource-windows-x86-5.13.1.exe`. In the installer, you can skip making an account, and all you need `Qt` > `Qt 5.13.1` > `MSVC 2017 32-bit`.
```
Comment (mig5) [inline BUILD.md]: @micahflee tiny nitpick: 'Install the Qt 5.13.0' should now be 'Install the Qt 5.13.1'. Maybe we could rephrase this to 'Install version 5.13.1 of Qt'.

File: BUILD.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Download and install Python 3.7.4 from https://www.python.org/downloads/release/
 
 You may also need to run the command `/Applications/Python\ 3.7/Install\ Certificates.command` to update Python 3.6's internal certificate store. Otherwise, you may find that fetching the Tor Browser .dmg file fails later due to a certificate validation error.
 
-Install Qt 5.13.0 for macOS from https://www.qt.io/offline-installers. I downloaded `qt-opensource-mac-x64-5.13.0.dmg`. In the installer, you can skip making an account, and all you need is `Qt` > `Qt 5.13.0` > `macOS`.
+Install Qt 5.13.0 for macOS from https://www.qt.io/offline-installers. I downloaded `qt-opensource-mac-x64-5.13.1.dmg`. In the installer, you can skip making an account, and all you need is `Qt` > `Qt 5.13.10` > `macOS`.
```
Comment (mig5) [inline BUILD.md]: 'Install Qt 5.13.1' (not 5.13.0) - also is '5.13.10' a typo?

**Comment by micahflee** [comment]
I'd totally be into including Tor 0.4.1.6, but I just checked and it's not in the latest Tor Browser alpha yet (9.0a6), which means the only way to use it right now is to build from source. Which we're going to need to start doing anyway for #953 -- but we haven't yet, so I'm fine with sticking with Tor 0.4.1.5 until the next release.
